docs/packages/omc3/workflow.md

```diff
@@ -1,7 +1,220 @@
 
-!!! todo
-    Description of a typical analysis workflow. With easy examples for first-timers.
+## The Analysis Workflow
 
-!!! todo
-    Also add smaller examples for individual entry-points, scripts and plotting.
+A typical workflow with `omc3` consists in performing analysis of measurement or simulation files, and eventually calculating corrections to apply.
+While `omc3` provides python modules to handle the different aspects of these tasks, it also provides entrypoint scripts to be called from the commandline.
+
+The first step in the workflow consists in a harmonic frequency analysis performed with the `harpy` module, while the second one is an optics analysis performed with the `measure_optics` module.
+The table below shows a general analysis workflow from BPM turn-by-turn measurements or simulation file to results, as well as the corresponding files at each step:
+
+| Workflow & Files       |                        |                        |                        |                                   |
+| :--------------------- | :--------------------: | :--------------------: | :--------------------: | --------------------------------: |
+| Turn-by-Turn BPM Data  | :material-arrow-right: | Frequency Spectra      | :material-arrow-right: | Various Lattice Optics Parameters |
+| SDDS files:  `*.sdds`  | :material-arrow-right: | TFS files: `*.lin[xy]` | :material-arrow-right: | TFS files: `*.tfs`                |
+
+In this page, we will go through the essential steps in preparing and performing an analysis, by going from start to finish with a simple example.
+In this walk-through, we will cover the use of the different entrypoints available to perform necessary steps.
+
+!!! example  ""
+    For our walk-through example, we will start from data obtained in `MAD-X` with the `TRACK` command for the 1023 turns in the LHC machine, with a scenario adapted from the 2018 configuration.
+    Changes from the nominal scenario in your simulation could be including errors tables, orbit bumps, speculative magnet errors, additional elements etc.
+    It is also easy for the reader to follow along if starting from measurements files.
+
+## Preparing Data for Analysis
+
+Analysis in `omc3` is mostly done from measurement data, but can also be done on simulated tracking data.
+While `omc3` codes can read data from many machine formats, we will for demonstration purposes first convert our data to the LHC's [SDDS][sdds] binary format.
+
+For this `omc3` provides the `tbt_converter` entrypoint.
+A typical use consists in specifying the location of your turn-by-turn measurement files, the data type of said files and the location in which to write the converted `SDDS` data.
+The formats supported by the converter are:
+
+- Machine formats: `lhc`, `iota` and `esrf`, or any machine using one of these.
+- Simulation formats: the `trackone` and `ptc` formats from tracking data from `MAD-X` or `PTC`.
+
+??? info "The `trackone` and `ptc` formats"
```

Comment:
Could we have a link to the script creating the trackone?

Reply:
I can add it similarly to how I put the example script in the `job_submitter` documentation, yes.